modules/simple_forms_api/spec/requests/v1/uploads_spec.rb

```diff
@@ -309,20 +309,27 @@
   end
 
   describe '#submit_supporting_documents' do
-    it 'renders the attachment as json' do
+    it 'renders the attachment as json for multiple form IDs' do
       clamscan = double(safe?: true)
       allow(Common::VirusScan).to receive(:scan).and_return(clamscan)
       file = fixture_file_upload('doctors-note.gif')
-      data = { form_id: '40-0247', file: }
 
-      expect do
-        post '/simple_forms_api/v1/simple_forms/submit_supporting_documents', params: data
-      end.to change(PersistentAttachment, :count).by(1)
+      # Define data for both form IDs
+      data_sets = [
+        { form_id: '10-10D', file: file },
+        { form_id: '40-0247', file: file }
```

Comment:
The linter is mad because you can write `{ file: file }` as `{ file: }`. It's weird but I guess that's the rule this codebase has adopted.